GL-js-hw-03/task-06.js

```diff
@@ -0,0 +1,18 @@
+const products = [
+  { name: 'Радар', price: 1300, quantity: 4 },
+  { name: 'Сканер', price: 2700, quantity: 3 },
+  { name: 'Дроид', price: 400, quantity: 7 },
+  { name: 'Захват', price: 1200, quantity: 2 },
+];
+
+const calculateTotalPrice = function(allProdcuts, productName) {
+  // вот никак не получается )))
```

Comment:
Так як у нас масив із об'єктами, спочатку потрібно знайти правильний об'єкт, а потім вже витягувати деталі по ключах
```suggestion
  function calculateTotalPrice(allProdcuts, productName) {
    const product = allProdcuts.find(item => item.name === productName);  
    return product.price * product.quantity;
};

```